s3/replication/manager/src/s3replicationmanager/manager_routes.py

```diff
@@ -135,16 +164,18 @@ async def list_jobs(request):
     url_ob = urlparse(url)
     query = parse_qs(url_ob.query)
 
+    _logger.debug('API: GET /jobs\n Query: {}'.format(query))
+
     progressing_jobs_obj = request.app['jobs_in_progress']
     all_jobs_obj = request.app['all_jobs']
 
-    # LOG.debug(
+    # _logger.debug(
```

Comment:
these logs can be removed if not required.

Reply:
Fixed